controlplane/kubeadm/internal/webhooks/kubeadm_control_plane.go

```diff
@@ -620,6 +620,13 @@ func (webhook *KubeadmControlPlane) validateCoreDNSVersion(oldK, newK *controlpl
 	if toVersion.Equals(fromVersion) {
 		return allErrs
 	}
+
+	// Skip validating if the CoreDNS migration library supports the upgrade if the skip CoreDNS annotation is set.
+	// If this annotation is set, KCP doesn't use the migration library.
```

Comment:
should it be just this?

```suggestion
	// Skip validating if the skip CoreDNS annotation is set. If set, KCP doesn't use the migration library.
```

Reply:
Fine fore me, thx!